api/apis/route_handler_test.go

```diff
@@ -815,7 +815,7 @@ var _ = Describe("RouteHandler", func() {
 			})
 
 			It("returns an error", func() {
-				expectUnprocessableEntityError("Invalid Route, FQDN does not comply with RFC 1035 standards")
+				expectUnprocessableEntityError("Invalid Route, {\"validationErrorType\":\"RouteFQDNValidationError\",\"message\":\"FQDN does not comply with RFC 1035 standards\"}")
```

Comment:
This assertion is brittle, since the JSON field order can change. Better to use a `ContainSubstring` matcher, or somehow json parse the message and assert against the parsed result. Maybe we need a specific matcher for these kind of validation messages?

Reply:
Hmm... probably best to change it still, unless you're going to remove it almost immediately. The last thing we need is more flakiness in tests. You can use the `SatisfyAll` matcher to apply a `HavePrefix` matcher and 2 `ContainSubstring` matchers, which will avoid the problem.